### ISSUE TITLE
Not Compatible with Dotnet Core

### ISSUE THREAD

**Issue by thefringeninja** [akordowski/Cake.Compression#12]
Getting the following error when trying to use this adding with Cake.CoreClr:
```
Could not find any assemblies compatible with .NETStandard,Version=v1.6.
Error: Cake.Core.CakeException: Failed to install addin 'Cake.Compression'.
```
This is probably because this addin is still being compiled against the full dotnet framework instead of netstandard. If you would like I can send a PR to fix this.

**Comment by akordowski** [comment]
The Addin use under the hood the SharpZip library, which is a .net lib. Just compiling the project with .netstandard might not solve the problem. But I will take a look on this and inform you.

**Comment by akordowski** [comment]
@thefringeninja As I expected the SharpZip lib is incompatible with .netstandard. If you are using the Zip functionality only, you can try with the Cake [ZipAliases](https://cakebuild.net/api/Cake.Common.IO/ZipAliases/) methods.

**Comment by thefringeninja** [comment]
https://www.nuget.org/packages/SharpZipLib/1.0.0-alpha2 looks compatible with netstandard1.6 now

In any case the zip aliases do not work with gzip compression

**Comment by git-man** [comment]
Yep, I built the Cake.Compression against the latest SharpZipLib (alpha) targetting netstandard2.0.

As an example I changed only the csproj (see below) it works perfectly. Maybe you could use it in way and bundle it to release? 

Would be cool. Besides that: Thanks for great work!

<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>Library</OutputType>
    <RootNamespace>Cake.Compression</RootNamespace>
    <AssemblyName>Cake.Compression</AssemblyName>
    <TargetFramework>netstandard2.0</TargetFramework>
    <FileAlignment>512</FileAlignment>
  </PropertyGroup>
  <ItemGroup>
	 <PackageReference Include="Cake.Core" Version="0.26.1" />
	 <PackageReference Include="Cake.Common" Version="0.26.1"/>
	 <PackageReference Include="SharpZipLib" Version="1.0.0-alpha2" />
  </ItemGroup>
</Project>

**Comment by akordowski** [comment]
I take a look.

**Comment by damianh** [comment]
Did you take a look?

**Comment by akordowski** [comment]
Yes, I took a look. I decided not to use the alpha version. But I saw there is a RC1 released yesterday. I have to test if it targets the netstandard and when the 1.0 is released I will update it.

**Comment by akordowski** [comment]
@damianh Thank you for the PR. I released the version 0.2.0 which supports now .netstandard2.0 and SharpZipLib 1.0.0-rc1.

**Comment by damianh** [comment]
Good stuff :)
